test/dotnet-pack.Tests/PackTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.DotNet.Tools.Compiler.Tests
+{
+    public class PackTests : TestBase
+    {
+        private readonly string _testProjectsRoot;
+
+        public PackTests()
+        {
+            _testProjectsRoot = Path.Combine(AppContext.BaseDirectory, "TestAssets", "TestProjects");
+        }
+
+        [Fact]
+        public void OutputsPackagesToConfigurationSubdir()
+        {
+            var root = Temp.CreateDirectory();
+
+            var testLibDir = root.CreateDirectory("TestLibrary");
+            var sourceTestLibDir = Path.Combine(_testProjectsRoot, "TestLibrary");
+
+            CopyProjectToTempDir(sourceTestLibDir, testLibDir);
+
+            var testProject = GetProjectPath(testLibDir);
+            var buildCommand = new PackCommand(testProject);
+            var result = buildCommand.Execute();
+            result.Should().Pass();
+
+            var outputDir = new DirectoryInfo(Path.Combine(testLibDir.Path, "bin", "Debug"));
```

Comment:
Just in this test - so we know we're using the passed in value as opposed to something the CLI sets up?